cpp/cppsdk/gsdk.h

```diff
@@ -27,6 +34,27 @@ namespace Microsoft
                 }
             };
 
+            class MaintenanceEvent
+            {
+            public:
+                std::string m_eventId;
+                std::string m_eventType;
+                std::string m_resourceType;
+                std::vector<std::string> m_resources;
+                std::string m_eventStatus;
+                tm m_notBefore;
+                std::string m_description;
+                std::string m_eventSource;
+                int m_durationInSeconds;
```

Comment:
Nit: This is a comment for throughout this codebase. Is int the right type for m_durationInSeconds? Should it be uint32_t?

Reply:
I'm changing this for the new maintenanceschedule, but leaving existing instances of int for the moment